src/libs/CurrencyUtils.ts

```diff
@@ -99,7 +99,7 @@ function convertToFrontendAmount(amountAsInt: number): number {
  * @param currency - IOU currency
  * @param shouldFallbackToTbd - whether to return 'TBD' instead of a falsy value (e.g. 0.00)
  */
-function convertToDisplayString(amountInCents: number, currency: string = CONST.CURRENCY.USD, shouldFallbackToTbd = false): string {
+function convertToDisplayString(amountInCents = 0, currency: string = CONST.CURRENCY.USD, shouldFallbackToTbd = false): string {
```

Comment:
Why this change?

Reply:
yea, we need a default value to avoid Nan, when we pass undefined values.